agent/src/jmh/java/com/code_intelligence/jazzer/instrumentor/BUILD.bazel

```diff
@@ -50,7 +51,7 @@ java_library(
     ],
     deps = [
         "//agent/src/main/java/com/code_intelligence/jazzer/instrumentor",
-        "@jazzer_jacoco//:jacoco_internal",
+        "@com_github_jacoco_jacoco//:jacoco_internal",
```

Comment:
With (soon to be deprecated, but pervasive) `WORKSPACE` files, repository names must be globally unique across a build and the first definition "wins". Since we can only compile against a properly patched version of JaCoCo, we should stick to a name prefixed with `jazzer`.

Reply:
Changed the name back to the original.